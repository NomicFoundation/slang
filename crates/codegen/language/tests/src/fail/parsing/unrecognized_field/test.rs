#![allow(unused_crate_dependencies)]

codegen_language_macros::compile!(Language(
    name = Foo,
    documentation_dir = "foo/bar",
    root_item = Bar,
    leading_trivia = Sequence([]),
    trailing_trivia = Sequence([]),
    versions = ["1.0.0"],
    sections = [Section(
        title = "Section One",
        topics = [Topic(
            title = "Topic One",
            items = [Token(
                name = Bar,
                definitions = [TokenDefinition(scanner = Atom("bar"))]
            )],
            unrecognized_field = true
        )]
    )]
));

fn main() {}
