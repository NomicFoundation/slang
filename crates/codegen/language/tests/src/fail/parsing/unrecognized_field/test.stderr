error: unexpected token
  --> src/fail/parsing/unrecognized_field/test.rs:18:13
   |
18 |             unrecognized_field = true
   |             ^^^^^^^^^^^^^^^^^^
